quicksort.py

```diff
@@ -0,0 +1,12 @@
+# Part of Cosmos by OpenGenus Foundation

```

Comment:
This file should be remove, since you already have the other same file on `sorting/quick_sort/quicksort.py`